db/app/models/articles.model.js

```diff
@@ -39,3 +39,10 @@ exports.addComment = (article_id, comment) => {
     return rows;
   });
 };
+
+exports.alterVotes = (article_id, inc_votes) => {
+return db.query( `UPDATE articles
+  SET votes = votes + $1
+  WHERE article_id = $2 RETURNING *;`, [inc_votes, article_id]).then(({rows})=> {
+    return rows
```

Comment:
We don't necessarily need to call fetchArticleById(article_id) to confirm whether the article Id is valid, as due to this WHERE clause, if the article_id does not exist, we will return an empty array. Instead (to avoid the promise.All syntax), we can instead just check the length of the row and throw the appropriate error if empty.